pkg/api/v1/types.go

```diff
@@ -428,6 +428,9 @@ type PersistentVolumeSource struct {
 	// ScaleIO represents a ScaleIO persistent volume attached and mounted on Kubernetes nodes.
 	// +optional
 	ScaleIO *ScaleIOVolumeSource `json:"scaleIO,omitempty" protobuf:"bytes,19,opt,name=scaleIO"`
+	// LocalVolume represents a local volume accessible only from one node
+	// +optional
+	LocalVolume *LocalVolumeSource `json:"localVolume,omitempty" protobuf:"bytes,20,opt,name=localVolume"`
```

Comment:
Yeah the proposal will need some updates after addressing the review comments on this PR.